### PR TITLE
visualize stow json

### PR DESCRIPTION
**Pull request by kimhc6028**
frame0000
Stow version of visualize_json.py

### CODE REVIEW

File: jsk_apc2016_common/python/jsk_apc2016_common/__init__.py [hunk 1]
```diff
@@ -5,13 +5,127 @@
...
+                if contents:
+                    obj = contents.pop()
+                    obj_img = object_imgs[obj]
+                    scale_h = 1. * max_obj_h / obj_img.shape[0]
+                    scale_w = 1. * max_obj_w / obj_img.shape[1]
+                    scale = min([scale_h, scale_w])
+                    obj_img = rescale(obj_img, scale)
+                    obj_h, obj_w = obj_img.shape[:2]
+                    x_max, y_max = x_min + obj_w, y_min + obj_h
+                    bin_region[y_min:y_max, x_min:x_max] = obj_img
```
Comment (wkentaro) [inline jsk_apc2016_common/python/jsk_apc2016_common/__init__.py]: why not reuse `jsk_apc2015_common.visualize_json` ?

Comment (kimhc6028) [inline jsk_apc2016_common/python/jsk_apc2016_common/__init__.py]: `jsk_apc2015_common.visualize_json` needs to be edited for a few lines to be reused. That is why I was thinking not to reuse :). I will edit those.

**Comment by kimhc6028** [comment]
frame0000
The change does not affect original functionality of visualize_json.

**Comment by wkentaro** [comment]
Closed via https://github.com/start-jsk/jsk_apc/pull/1133

**Pull request closed without merging**